src/test/java/org/folio/ebsconet/TestUtils.java

```diff
@@ -1,13 +1,6 @@
 package org.folio.ebsconet;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
-import com.fasterxml.jackson.databind.ObjectMapper;
 import org.apache.commons.io.IOUtils;
-import org.hamcrest.Description;
-import org.hamcrest.DiagnosingMatcher;
-import org.skyscreamer.jsonassert.JSONCompare;
-import org.skyscreamer.jsonassert.JSONCompareMode;
-import org.skyscreamer.jsonassert.JSONCompareResult;
 
```

Comment:
These are not used so deleted

Reply:
In case it wasn't affected  by dependenies update, such changes should be done in separate commits

Reply:
Ok, Fixed